votingpool/pool.go

```diff
@@ -55,6 +64,37 @@ type Pool struct {
 	namespace    walletdb.Namespace
 }
 
+// PoolAddress represents a voting pool P2SH address, generated by
+// deriving public HD keys from the series' master keys using the given
+// branch/index and constructing a M-of-N multi-sig script.
+type PoolAddress interface {
+	fmt.Stringer
```

Comment:
Same deal with the required `Stringer` here as for the `Series` interface.  I don't think it should be required by the `PoolAddress` interface.